client/src/shared/components/ChoiceDiagram/ChoiceDiagram.module.css

```diff
@@ -0,0 +1,10 @@
+.container {
+  width: 500px;
```

Comment:
This is temporary. We will want to make this a function of how much shit is in the canvas. For now this gets it to show up, but I'll be revisiting this.